ONSdigital/census-rm-action-worker
# Port integration test from action scheduler

# Motivation and Context:
There were no integration tests to make sure the poller was working.

# What has changed?
Took the main integration test from action scheduler and ported it over into this project.

# How to test?
Build and run the tests. Run the acceptance tests. There should be zero regression.

# Links:
Trello: https://trello.com/c/462ZhSzd
